requirements.txt

```diff
@@ -1,2 +1,2 @@
 click==6.2
-Jinja2>=2.8
+Jinja2==2.8
```

Comment:
Well, since (1) this is unrelated to this PR and (2) I guess we shouldn't pin this (because this may cause dependency issues for people using this lib) I'll revert this change.